src/contracts/digital-twin/container.ts

```diff
@@ -1476,6 +1458,32 @@ export class Container extends Logger {
             false,
           );
         }
+      }));
+
+      await Throttle.all(accessP);
+
+      // /////////////////// check if only remaining property is 'type', cleanup if that's the case
```

Comment:
Whats `///////////////////` good for?

Reply:
As the code blocks are huuuuuge, we inserted comments like `//////// right-aligned-text` to separate the blocks visually. Eslint didn't like `//////`, so we turned it into `// ////`.
So it's just to separate the sections visually.